widgets/post-carousel/js/carousel.js

```diff
@@ -106,7 +110,51 @@ jQuery( function ( $ ) {
 				}
 			} );
 
+		} );
 
+		// Keyboard Navigation of carousel navigation.
+		$( document ).on( 'keydown', '.sow-carousel-navigation a', function( e ) {
+			if ( e.keyCode != 13 && e.keyCode != 32 ) {
+				return;
+			}
+			e.preventDefault();
+			$( this ).click();
+		} );
+
+		// Keyboard Navigation of carousel items.
+		$( document ).on( 'keyup', '.sow-carousel-item', function( e ) {
+			// Ensure left/right key was pressed
+			if ( e.keyCode != 37 && e.keyCode != 39 ) {
+				return;
+			}
+
+			var $wrapper =  $( this ).parents( '.sow-carousel-wrapper' ),
+			$items = $wrapper.find( '.sow-carousel-items' ),
```

Comment:
For clarity, I always prefer it if these are indented one level. Not sure what the WordPress style guide says here though.

Reply:
The Style guide actually recommends the spacing also. 😞